apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js

```diff
@@ -116,6 +117,128 @@ const VAT_TYPE = {
+ * @property {number} cdiPermDocQnt (integer, optional)    Содержит информацию о расчетных документах, ожидающих разрешения на проведение операции. Количество документов
+ * @property {number} cdiPermDocSum (number, optional)    Содержит информацию о расчетных документах, ожидающих разрешения на проведение операции. Сумма документов
+ * @property {string} closeDate (string, optional)    Дата закрытия счета
+ * @property {string} comment (string, optional)    Примечание
+ * @property {boolean} creditBlocked (boolean, optional)    Признак полной блокировки счета по кредиту
+ * @property {string} creditBlockedBeginDate (string, optional)    Дата начала действия ограничения
+ * @property {string} creditBlockedCause (string, optional)    Основание ареста
+ * @property {string} creditBlockedEndDate (string, optional)    Дата снятия ограничения
+ * @property {string} creditBlockedInitiator (string, optional)    Наименование органа, наложившего арест
+ * @property {string} creditBlockedTaxAuthorityCode (string, optional)    Код налогового органа, наложившего арест
+ * @property {string} currencyCode (string, optional)    Цифровой код валюты счета
+ * @property {boolean} dbo (boolean, optional)    Признак обслуживания в ДБО
+ * @property {boolean} debitBlocked (boolean, optional)    Признак полной блокировки счета по дебету
+ * @property {string} debitBlockedBeginDate (string, optional)    Дата начала действия ограничения
+ * @property {string} debitBlockedCause (string, optional)    Основание ареста
+ * @property {string} debitBlockedEndDate (string, optional)    Дата снятия ограничения
+ * @property {string} debitBlockedInitiator (string, optional)    Наименование органа, наложившего арест
+ * @property {string} debitBlockedTaxAuthorityCode (string, optional)    Код налогового органа, наложившего арест
+ * @property {number} minBalance (number, optional)    Минимальный поддерживаемый (неснижаемый) остаток на счете
+ * @property {string} mode (string, optional)    Режим работы счета = [STANDART, FORBIDDEN_RECEIVING, ONLY_RECEIVING]
+ * @property {string} name (string, optional)    Наименование счета
+ * @property {boolean} notDelay (boolean, optional)    Признак возможности проведения неотложных платежей
+ * @property {string} number (string, optional)    Номер счета (20 сиволов)
+ * @property {string} openDate (string, optional)    Дата открытия счета
+ * @property {number} overdraft (number, optional)    Сумма общего лимита овердрафта в валюте счета
+ * @property {boolean} passive (boolean, optional)    Признак пассивности счета
+ * @property {string} state (string, optional)    Состояние счета = [OPEN, BLOCKED, CLOSED]
+ * @property {string} type (string, optional)    Тип счета = [assuranceRegistration, calculated, transit, specialTransit, budget, loan, deposit]
+ * @property {boolean} urgent (boolean, optional)    Признак возможности проведения срочных платежей
+ * @example
+ * "accounts":[
```

Comment:
We have an object documentation here.
```suggestion
```

Reply:
accounts (Array[Account], optional) | Счета организации клиента, доступные партнеру,
-- | --


this is from the fintech documentation

Reply:
You describing an entity here which is an object, and you're giving an example of an array.
Example should correspond to entity, you are describing.
It does not matter what is at SBBOL side, because we are documenting our side

Reply:
oh, I misunderstood. If you are talking about the fact that the example is not needed, since there is already a described object